code/datums/components/slippery.dm

```diff
@@ -25,6 +25,9 @@
 		COMSIG_ATOM_ENTERED = .proc/Slip_on_wearer,
 	)
 
+	/// The connect_loc_behalf component for the holder_connections list.
+	var/datum/weakref/holder_connect_loc_behalf
```

Comment:
Have you put any thought into the signals thing I mentioned to you?

Reply:
yeah but it'd require a lot of new signals for everything to use it properly. I don't want to do it for this one isolated case because then it'd be inconsistent.

If I do it, I'd do it for all of them but that's a lot of extra work. It's something that can be done for the future.

Reply:
good enough for me